e2e/e2e_test.go

```diff
@@ -52,6 +52,7 @@ func init() {
 	flag.StringVar(&fileSystemName, "filesystem", "myfs", "CephFS filesystem to use")
 	flag.StringVar(&clusterID, "clusterid", "", "Ceph cluster ID to use (defaults to `ceph fsid` detection)")
 	flag.StringVar(&nfsDriverName, "nfs-driver", "nfs.csi.ceph.com", "name of the driver for NFS-volumes")
+	flag.BoolVar(&operatorDeployment, "operator-deployment", false, "test running on deployment via operator")
```

Comment:
```suggestion
flag.BoolVar(&operatorDeployment, "operator-deployment", false, "deploy ceph-csi using the operator")
```

Reply:
I am missing why this is needed. But maybe that becomes clear in a next commit (could be squash together)?